2.6.0/spec/orders.md

```diff
@@ -644,7 +650,7 @@ The date and time when the order is expected to be delivered. This is an estimat
 ### spSubscriptionId
 
 * Data format: [spSubscriptionId](../common/dataformats.md#spsubscriptionid)
-* Required in response, if set
+* Required
 
 ### subscriptionId
```

Comment:
The line with `Required for the operations SUSPEND, RESUME, MODIFY and CHANGE` below is no longer needed.